packages/cli/src/models/config/ConfigManager.ts

```diff
@@ -38,6 +40,7 @@ const ConfigManager = {
       await ZWeb3.checkNetworkId(network.networkId);
       const txParams = {
         from: ZWeb3.toChecksumAddress(from || artifactDefaults.from || (await ZWeb3.defaultAccount())),
+        ...pick(artifactDefaults, ['gas', 'gasPrice']),
```

Comment:
Can we change this so these keys are not set if their value is empty? I want to avoid a case where we merge `txParams` using an object splat, which would end up overriding an existing value with `undefined` if we're not careful.

Reply:
As an example, in the following code, if `txParams = { gasPrice: undefined }`, the default `10e9` value would get overwritten with `undefined` (which is not the desired behaviour).

```
function foo(txParams) {
  const txParamsDefaults = { gasPrice: 10e9 }
  const txParamsToUse = { ...txParamsDefaults, ...txParams }
  ...
}
```

Reply:
This is currently not an issue due to how these values are created (empty entries are removed), but I addressed this nonetheless in the style suggested in https://github.com/OpenZeppelin/openzeppelin-sdk/issues/1231.

I also created https://github.com/OpenZeppelin/openzeppelin-sdk/issues/1346 to track this general issue.